sdk/migrationdiscoverysap/Azure.ResourceManager.MigrationDiscoverySap/tests/Tests/MigrationSapDiscoveryTests.cs

```diff
@@ -146,13 +145,16 @@ public async Task PostImportEntities(ArmClient client, SapDiscoverySiteResource
         var operationStatusObj = JObject.Parse(opStatus.Content.ToString());
         var inputExcelSasUri = operationStatusObj?["properties"]?["discoveryExcelSasUri"].ToString();
 
-        //Upload here
-        using (FileStream stream = File.OpenRead(excelPathToImport))
+        if (SessionEnvironment.Mode != RecordedTestMode.Playback)
```

Comment:
```suggestion
        if (SessionEnvironment.Mode == RecordedTestMode.Record)
```
We only want this during recording, there is another model `None`

Reply:
`none` is `live`, which needs to skip this as well